### PR TITLE
Photo upload

### PR DESCRIPTION
**Pull request by podmar**
Hi Lina, I created the photoupload route, without saving the link in mongo just yet. My postman is acting out while attaching the file - if it works for you, could you please test it? Otherwise I find a solution for the postman issue tomorrow or later tonight. Take care, Marta

### CODE REVIEW

File: server/src/server.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import express from "express";
+import express, {Request, Response} from "express";
```
Comment (linablum) [inline server/src/server.ts]: I don't think we need to import Request/Response here, but if you need it for some testing it's alright

File: server/src/controllers/usersController.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from 'express';
+// import { userModel } from "../models/userModel";
+import { v2 as cloudinary } from "cloudinary";
+
```
Comment (linablum) [inline server/src/controllers/usersController.ts:3]: Can you delete some code which is doubled and commented out, cause I think it is good to have it cleaner from the beginning than clean up at the end

File: server/src/models/userModel.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        unique: true,
+    },
+    location: {
+        type: String,
+    },
+    tech_offer:{
+        type:Array,
+        }, 
+    y_xp:{
+        type:Number,
```
Comment (linablum) [inline server/src/models/userModel.ts]: I think you said you already changed the model and took that stuff for offer/req out (?), just as reminder I commented

File: server/src/server.ts (hunk 1 above)
Comment (podmar) [reply server/src/server.ts]: let's remove!

File: server/src/controllers/usersController.ts (hunk 2 above)
Comment (podmar) [reply server/src/controllers/usersController.ts:3]: there was no doubling, just the function saving to mongoose. But I agree it's better not to have code that is not being in use, removed all the comments.

File: server/src/models/userModel.ts (hunk 3 above)
Comment (podmar) [reply server/src/models/userModel.ts]: I have realised I confused it after the PR, now its adjusted to what we discussed :)

**Pull request merged**